catalogue/webapp/components/Work/Work.tsx

```diff
@@ -66,7 +67,8 @@ const Work: FunctionComponent<Props> = ({
   });
 
   const isInArchive = work.parts.length > 0 || work.partOf.length > 0;
-
+  // added condition isInArchive as its a hook as we only want to trigger in that view
+  useHotjar(globalContextData.toggles, isInArchive);
```

Comment:
By doing it like this - I wonder what the point is of having it as a toggle?

To simplify we could also have the `useHotjar` take no arguments, and just:`
```ts
if (isInArchive) {
  useHotjar()
}
```

Reply:
got a point there, i would still have to put the condition inside the function as eslint complains about conditions for functional components.

Reply:
> complains about conditions for functional components.

For function components or for hooks?

Reply:
hooks, but we talked about changing solution to useState. nice !